docs/tutorial/tutorial-0.rst

```diff
@@ -159,6 +159,26 @@ and without the need to re-install Python.
       C:\...>cd beeware-tutorial
       C:\...>py -m venv beeware-venv
       C:\...>beeware-venv\Scripts\activate.bat
+      
+    .. code-block:: powershell
+     
+      PS C:\...>md beeware-tutorial
+      PS C:\...>cd beeware-tutorial
+      PS C:\...>py -m venv beeware-venv
+      PS C:\...>beeware-venv\Scripts\activate.ps1       
```

Comment:
This is a good point that the `activate.bat` script won't work too well in PowerShell.

However, the `beeware-venv\Scripts\activate` script runs in both; so, we shouldn't an additional code-block.

Reply:
Tested to make sure "beeware-venv\Scripts\activate" works for both Command Prompt and PowerShell and it works. I took out the second block and I updated the language.